MSAL/test/integration/native_auth/end_to_end/sign_up/MSALNativeAuthSignUpUsernameAndPasswordEndToEndTests.swift

```diff
@@ -25,23 +25,23 @@
 import XCTest
 import MSAL
 
-final class MSALNativeAuthSignUpUsernameAndPasswordEndToEndTests: MSALNativeAuthEndToEndBaseTestCase {
-    private let usernamePassword = ProcessInfo.processInfo.environment["existingPasswordUserEmail"] ?? "<existingPasswordUserEmail not set>"
-    private let password = ProcessInfo.processInfo.environment["existingUserPassword"] ?? "<existingUserPassword not set>"
-    private let attributes = ["age": 40]
-    
-    // Hero Scenario 2.1.1. Sign up - with Email verification as LAST step (Email & Password)
+final class MSALNativeAuthSignUpUsernameAndPasswordEndToEndTests: MSALNativeAuthEndToEndPasswordTestCase {
```

Comment:
Does this class still need to extend `MSALNativeAuthEndToEndPasswordTestCase` and not `MSALNativeAuthEndToEndBaseTestCase.swift`?

Reply:
Done